tooling/api/src/updater.ts

```diff
@@ -29,6 +29,14 @@ interface UpdateResult {
   shouldUpdate: boolean
 }
 
+async function onUpdaterEvent(
```

Comment:
Ideally this would be one object `updater` which has methods and is an event emitter, so people can use the proper `.addEventListener('<updater-event>', () => {})` JavaScript idiomatic style. But for the same of not breaking this this is pretty good 👍🏻

Reply:
let's add that to the v2 todo list :joy: